### ISSUE TITLE
Error when both tracked interfaces go down & then up

### ISSUE THREAD

**Issue by shamil** [acassen/keepalived#69]
If monitored interface(s) on both servers goes down and then up, keepalived is not bringing up the Virtual IP and just stuck with the error:

```
Keepalived_vrrp[18056]: cant do IP_DROP_MEMBERSHIP errno=Cannot assign requested address (99)
```

In order to make it work again, only restarting the service is helping. What can be the issue?

I'm using `unicast_peer`.
Here is the config I'm using

```
! secondary
vrrp_instance LB_1 {
    interface eth0
    virtual_router_id 122
    priority 101

    track_interface {
        ! this is private lan interface    
        bond0
    }

    virtual_ipaddress {
        x.x.x.10/27 dev eth0
    }

    unicast_peer {
        10.0.0.122
        10.0.0.123
    }
}

! primary
vrrp_instance LB_2 {
    interface eth0
    virtual_router_id 123
    priority 100

    track_interface {
        ! this is private lan interface
        bond0
    }

    virtual_ipaddress {
        x.x.x.20/27 dev eth0
    }

    unicast_peer {
        10.0.0.122
        10.0.0.123
    }
}
```

**Comment by ersushantsood** [comment]
I am using 1.2.13 keepalived on RHEL 7 and i am getting the same issue . Can you please let me know in which version this fix is available.

**Comment by pqarmitage** [comment]
The best version to try is 1.2.22. In addition to many other fixes, 1.2.22 also has a fix for when communications is lost between servers without the interfaces going down.
